test/services/katello/host_trace_manager_test.rb

```diff
@@ -0,0 +1,65 @@
+require 'katello_test_helper'
+
+unless Katello.with_remote_execution?
+  class JobInvocationComposer
```

Comment:
Can you confirm that these tests pass when we call into this block? Simplest way would be to just comment out the `unless` and `end` that way, when this reaches downstream where we'll enter into this block, that the test still pass!

Reply:
They do still pass when I comment out `unless` and `end`

```ruby
[vagrant@centos7-katello-devel-stable foreman]$ ktest ../katello/test/services/katello/host_trace_manager_test.rb 
/home/vagrant/foreman/lib/foreman.rb:8: warning: already initialized constant Foreman::UUID_REGEXP
/home/vagrant/foreman/lib/foreman.rb:8: warning: previous definition of UUID_REGEXP was here
-- execute("SET CONSTRAINTS ALL DEFERRED;")
WARNING:  SET CONSTRAINTS can only be used in transaction blocks
   -> 0.0005s
Run options: --seed 50217

# Running:

...

Finished in 2.459055s, 1.2200 runs/s, 6.0999 assertions/s.

3 runs, 15 assertions, 0 failures, 0 errors, 0 skips
Coverage report Rcov style generated for Unit Tests to /home/vagrant/foreman/coverage/rcov
Coverage report generated for Unit Tests to /home/vagrant/foreman/coverage. 0 / 0 LOC (100.0%) covered.
[vagrant@centos7-katello-devel-stable katello]$ git diff
diff --git a/test/services/katello/host_trace_manager_test.rb b/test/services/katello/host_trace_manager_test.rb
index 8cc693f..397dbbf 100644
--- a/test/services/katello/host_trace_manager_test.rb
+++ b/test/services/katello/host_trace_manager_test.rb
@@ -1,8 +1,8 @@
 require 'katello_test_helper'
 
-unless Katello.with_remote_execution?
+#unless Katello.with_remote_execution?
   class JobInvocationComposer
-  end
+#  end
 end
 
 module Katello
```